src/main/java/org/embulk/util/config/ConfigMapper.java

```diff
@@ -103,6 +97,21 @@
         return value;
     }
 
+    private static String buildExceptionMessage(final Exception ex, final Class<?> taskType) {
```

Comment:
could we add unit test for this?

Reply:
Added in ed55c042af113aef50191393c8341bbf38c0645f